docs/medical-api/api-reference/fhir/consolidated-data-query-post.mdx

```diff
@@ -11,6 +11,10 @@ Once the data is consolidated and ready for consumption, a [Webhook request](/me
 is sent to [your configured URL](/home/api-reference/settings/post-settings) containing the Patient's
 data in FHIR-compliant format, based on the [parameters](#query-params) used to trigger the query.
 
+You'll be able to query multiple times for the same Patient which will be added to the queue and
+processed in order. If the parameters are the same as a previous query that is still processing,
+the new query will be ignored.
```

Comment:
nit: "...and you'll get the status and request ID of the previous one."

Thinking from the cx's perspective, it would be good to know whether a request to Metriport's API results in a new action or was it ignored and which one is running instead.